config/routes.rb

```diff
@@ -1,3 +1,5 @@
 Spree::Core::Engine.add_routes do
   # Add your extension routes here
+  post '/khalti', :to => "khalti#payment", :as => :khalti_payment
+  get '/khalti/payment-config', :to => "khalti#khalti_payment_config", :as => :khalti_payment_config, format: :json
```

Comment:
Should this have been `get '/khalti/payment_config'` instead? It's not in snake case right now.